src/main/java/unrc/dose/User.java

```diff
@@ -1,19 +1,39 @@
-/*
-* == Schema Info
-*
-* Table name: users
-*
-*  id              :integer(11)    not null, primary key
-*  username        :varchar(20)    not null,
-*  password        :varchar(20)    not null,
-*  email_address   :varchar(50)    not null,
-*  admin           BOOLEAN not null default 0
-*
-**/
+
 
 package unrc.dose;
 
 import org.javalite.activejdbc.Model;
+import org.javalite.activejdbc.LazyList;
 
 public class User extends Model {
+	static final String  PSW_RESET = "password reset";
+	static final String  PSW_NOT_RESET = "the email is not registered in the DB";
```

Comment:
These attributes should be private, unless they are used outside this class, but since there is no documentation attached I can't know if they will.